app/templates/email_templates/email_response_file.html

```diff
@@ -2,9 +2,19 @@
     {% if release_public_links and not release_private_links %}
```

Comment:
Can this file be refactored to reduce the amount of duplicated code and the number of if statements.